node/tests/TestUtilities.ts

```diff
@@ -295,23 +306,33 @@ export class RedisCluster {
         });
     }
 
+    public static connectToCluster(port: [string, number][]): RedisCluster {
```

Comment:
```suggestion
    public static connectToCluster(address: [string, number][]): RedisCluster {
```

Reply:
are we connecting to the cluster (e.g. creating a client), or just creating an cluster object ?
if not, change to initFromExistingCluster